src/woo_search/search_index/documents/publications.py

```diff
@@ -35,7 +35,7 @@ class Document(ES_Document):
     officiele_titel: M[str] = mapped_field(Text(required=True))
     verkorte_titel: M[str] = mapped_field(Text())
     omschrijving: M[str] = mapped_field(Text())
-    creatiedatum: M[datetime] = mapped_field(Date(required=True))
+    creatiedatum: M[date] = mapped_field(Date(format="yyyy-MM-dd", required=True))
```

Comment:
From checking the elasticsearch_dsl code, this format makes sure it's converted to a date by the library (`data = data.date()`) if `data` is a string, but is this needed if date is of type `date`?

Reply:
I am sorry but I don't understand your question here.

Reply:
I'm wondering why format="yyyy-MM-dd" is now specified here, if I check the es_dsl source code this should not be needed if we're passing `date` objects (https://github.com/elastic/elasticsearch-dsl-py/blob/c063cfaaa22aeb06fa49ef0701aaae007920e3f5/elasticsearch_dsl/field.py#L319)

Reply:
![image](https://github.com/user-attachments/assets/419202a2-5d12-45dc-bfbb-5fcda5a6b192)

In the documentation it is explicitly stated to do it this way.